bootstrap.sh

```diff
@@ -69,14 +69,53 @@ cd google-cloud-java
 git add pom.xml
 git commit -am 'feat: create aggregator pom'
 
+num_modules="$(wc -l < ../../repos.txt)"
+#echo "${num_modules}"
```

Comment:
Not needed?

Reply:
Yes, I'll remove these comments before merging.